tests/unit/test_serializer.py

```diff
@@ -268,6 +292,26 @@ def _get_test_parameter_by_id(default_values):
             default_value=default_values[15],
             annotations={},
         ),
+        17: ParameterMetadata(
+            display_name="enum_data",
+            type=type_pb2.Field.TYPE_ENUM,
+            repeated=False,
+            default_value=default_values[16],
+            annotations={
+                "ni/type_specialization": TypeSpecialization.Enum.value,
+                "ni/enum.values": "{'PURPLE': 0, 'ORANGE': 1, 'TEAL': 2, 'BROWN': 3}",
```

Comment:
Why is this using single quotes for the JSON key names? JSON strings don't support single quotes.

Reply:
Hmm, this appears incorrect. Switched to using `"` which coincidentally removed the need for `.replace("'", '"')` I used in `json.load`.